### ISSUE TITLE
When running Hyper through the Run Box on Windows, Hyper shows a black screen

### ISSUE THREAD

**Issue by harshpatel0** [vercel/hyper#5172]
When running Hyper through the Run Box, Hyper runs but doesn't show the Command Prompt on Windows

![image](https://user-images.githubusercontent.com/58038014/102449426-46391800-4045-11eb-96e5-42bfbf847dcf.png)

**Comment by apex-blaze** [comment]
Yeah , same problem here. Even updated to latest version , still doesn't works. Mine not even showing pink cursor!!!

**Comment by maclermo** [comment]
Same issue here. My fix was to populate the "shell" field in the settings (.hyper.js). After restarting the program, I was prompted with bash in my case. You can set it up with bash.exe, cmd.exe or powershell.exe. See the example in the settings file.

**Comment by apex-blaze** [comment]
> Same issue here. My fix was to populate the "shell" field in the settings (.hyper.js). After restarting the program, I was prompted with bash in my case. You can set it up with bash.exe, cmd.exe or powershell.exe. See the example in the settings file.

![image](https://user-images.githubusercontent.com/59483233/103134096-9a429d00-46d4-11eb-8f3a-c8de35b69a47.png)
I did as you said , but replacing git-cmd with git-bash  in 'shell' field and then reloading application opens blank hyper screen and an ordinary bash shell , which I don't want . I want to run specifically Hyper's Terminal which shows black screen sometimes and sometimes it runs good (Kinda moody!!). I tried to open its DEV Tools but it also shows Blank white screen. I installed no extra plugins for Hyper. Still can't find what's the cause!!!
I would appreciate it ,if you explain steps if you got the solution!!!!
